### PR TITLE
Updated for 4.40

### PR DESCRIPTION
**Pull request by SameerK-MSFT**
Fixes # 
Updated Changelog

**Changes proposed in this request**
NA

**Testing**
NA

**Performance impact**
NA

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,18 @@
+4.40.0
+==========
+### New Features:
+** Hybrid SPA is now production ready!! [2920]( https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920)
```
Comment (gladjohn) [inline changelog.txt:4]: the space before the hyperlink can be removed

**Review by gladjohn — APPROVED** [review]
looks good

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+** [Hybrid SPA Sample](https://aka.ms/msal-net/hybrid-spa-sample)
+** Allow POP token envelope to be created externally [3059](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3059)
+** Remove obsolete MATS telemetry. [3043](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3043)
+** Windows Form are referenced only in Desktop [3071](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/c240dad04ead8cb7a729ffcb5d23f0b02bf582b5)
```
Comment (trwalke) [inline changelog.txt:8]:
```suggestion
** Windows Forms are referenced only in Desktop [3071](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/c240dad04ead8cb7a729ffcb5d23f0b02bf582b5)
```

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,18 @@
...
+### New Features:
+** Hybrid SPA is now production ready!! [2920]( https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920)
+** [Hybrid SPA Sample](https://aka.ms/msal-net/hybrid-spa-sample)
+** Allow POP token envelope to be created externally [3059](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3059)
+** Remove obsolete MATS telemetry. [3043](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3043)
+** Windows Forms are referenced only in Desktop [3071](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/c240dad04ead8cb7a729ffcb5d23f0b02bf582b5)
+** Log clarification in several places.
+
+### Bug Fixes:
+** WAM can hide the windows (account picker and plugin browser) behind the app [2901](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2901)
```
Comment (pmaytak) [inline changelog.txt:12]: This can be removed. It was fixed in 4.37.

File: changelog.txt [hunk 4]
```diff
@@ -1,3 +1,18 @@
+4.40.0
+==========
+### New Features:
+** Hybrid SPA is now production ready!! [2920]( https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920)
+** [Hybrid SPA Sample](https://aka.ms/msal-net/hybrid-spa-sample)
+** Allow POP token envelope to be created externally [3059](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3059)
+** Remove obsolete MATS telemetry. [3043](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3043)
+** Windows Forms are referenced only in Desktop [3071](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/c240dad04ead8cb7a729ffcb5d23f0b02bf582b5)
+** Log clarification in several places.
```
Comment (pmaytak) [inline changelog.txt:9]: What is this in reference to? Add a link to related issue or PR.

File: changelog.txt [hunk 5]
```diff
@@ -1,3 +1,18 @@
+4.40.0
+==========
+### New Features:
+** Hybrid SPA is now production ready!! [2920]( https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920)
+** [Hybrid SPA Sample](https://aka.ms/msal-net/hybrid-spa-sample)
+** Allow POP token envelope to be created externally [3059](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3059)
+** Remove obsolete MATS telemetry. [3043](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3043)
```
Comment (pmaytak) [inline changelog.txt:7]:
```suggestion
**Remove obsolete telemetry (MATS) code** to improve performance and stability. [3043](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3043)
```

File: changelog.txt [hunk 6]
```diff
@@ -1,3 +1,18 @@
+4.40.0
+==========
+### New Features:
+** Hybrid SPA is now production ready!! [2920]( https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920)
+** [Hybrid SPA Sample](https://aka.ms/msal-net/hybrid-spa-sample)
```
Comment (pmaytak) [inline changelog.txt:5]:
```suggestion
**Authorization Code for Single Page Applications (SPA) feature is now generally available. `WithExperimentalFeatures()` is no longer required when calling `WithSpaAuthorizationCode()`. See [2920](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2920), [wiki](https://aka.ms/msal-net/spa-auth-code), and (sample)[https://aka.ms/msal-net/hybrid-spa-sample].
```

File: changelog.txt [hunk 7]
```diff
@@ -1,3 +1,18 @@
...
+** Allow POP token envelope to be created externally [3059](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3059)
+** Remove obsolete MATS telemetry. [3043](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3043)
+** Windows Forms are referenced only in Desktop [3071](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/commit/c240dad04ead8cb7a729ffcb5d23f0b02bf582b5)
+** Log clarification in several places.
+
+### Bug Fixes:
+** WAM can hide the windows (account picker and plugin browser) behind the app [2901](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/2901)
+** Allow res:// error pages to be displayed in embedded webview [3083](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3083)
+** MSAL Logs are not clear when regional is enabled and tokens are acquired from the cache [3073](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3073)
```
Comment (trwalke) [inline changelog.txt:14]:
```suggestion
** MSAL Logs are now more clear when regional is enabled and tokens are acquired from the cache [3073](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3073)
```

File: changelog.txt (hunk 4 above)
Comment (SameerK-MSFT) [reply changelog.txt:9]: There are two places. Added refs

**Review by gladjohn — APPROVED** [review]

**Review by bgavrilMS — APPROVED** [review]

**Pull request merged**